docs/notebooks/point_clouds.ipynb

```diff
@@ -13,30 +13,28 @@
   },
```

Comment:
<p>Please remove this artifact.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/189/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
thanks for the catch!